visualize_weather.py

```diff
@@ -100,6 +100,5 @@
 plt.ylim(-15, 111)
 plt.grid(False)
 
-plt.title('Philadelphia, PA\'s weather, July 2014 - June 2015\n\n', fontsize=20)
-
+plt.title('Houston\'s weather, July 2014 - June 2015\n\n', fontsize=20)
 plt.savefig('output/philadelphia-weather-july14-june15.png')
```

Comment:
Let's update the name of the output file as well
```suggestion
plt.savefig('output/houston-weather-july14-june15.png')
```